lib/instrumental/agent.rb

```diff
@@ -81,6 +82,9 @@ def initialize(api_key, options = {})
       @queue = Queue.new
 
       setup_cleanup_at_exit if @enabled
+
+      Metrician.activate(self)
```

Comment:
Probably we should have an opt-out for this for existing customers, and only require & activate if that (on by default) options is enabled.